.github/workflow/csi_operator_ci.yaml

```diff
@@ -0,0 +1,226 @@
+name: Operator test
+
+env:
+  OP_TEST_DEBUG: 1
+  OP_TEST_CONTAINER_OPT: "-t"
+on:
+  push:
+    pull_request:
+      - develop
+jobs:
+  operator_image_build:
+   name: "set env"
+   runs-on: ubuntu-latest
+   steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+   outputs:
+     operator_image: "csiblock1/ibm-block-csi-operator-amd64:new"
+
+  prepare_env:
+   name: "prepare_env"
+   needs: operator_image_build
+   runs-on: ubuntu-latest
+   steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+      - name: Install dependencies
+        run: python -m pip install --upgrade pip yq
+      - uses: FranzDiebold/github-env-vars-action@v2
+      - name: prepare_env
+        id: environment_setup
+        env:
+          operator_image: ${{ needs.operator_image_build.outputs.operator_image }}
+        run: |
+          build/ci/prepare_env.sh
+   outputs:
+     current_operator_image: "${{ steps.environment_setup.outputs.current_operator_image }}"
+     wanted_operator_image: "${{ steps.environment_setup.outputs.wanted_operator_image }}"
+     csv_file: "${{ steps.environment_setup.outputs.csv_file }}"
+     community_operators_path: "${{ steps.environment_setup.outputs.community_operators_path }}"
+     upstream_community_operators_path: "${{ steps.environment_setup.outputs.upstream_community_operators_path }}"
+     repository_path: "${{ steps.environment_setup.outputs.repository_path }}"
+
+  CI-csi-kiwi:
+    name: "kiwi / Full operator test"
+    needs: prepare_env
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: update_csv
+        run: |
+          build/ci/update_csv.sh
```

Comment:
could you open a ticket to reduce the code duplication, and write its number here?

Reply:
done